frame/assets/src/lib.rs

```diff
@@ -375,7 +375,6 @@ pub mod pallet {
 		pub accounts: Vec<(T::AssetId, T::AccountId, T::Balance)>,
 	}
 
-	#[cfg(feature = "std")]
```

Comment:
dito and more below.

Reply:
all done in: https://github.com/paritytech/substrate/pull/14108/commits/8a21eb158b0c03f2213fd8dfcafcaa9d0710397b